src/models/subscription.test.ts

```diff
@@ -1,289 +1,55 @@
-				}));
-			});
-		});
-		describe("findOneForGitHubInstallationId", ()=>{
-			it("should get for cloud record when gitHubAppId not present", async ()=>{
-				const record = await Subscription.findOneForGitHubInstallationId(
-					GITHUHB_INSTALLATION_ID
-				);
-				expect(record).toEqual(expect.objectContaining({
-					jiraHost,
-					jiraClientKey: "myClientKey",
-					gitHubInstallationId: GITHUHB_INSTALLATION_ID,
-					gitHubAppId: null
-				}));
-			});
-			it("should get correct github server app", async ()=>{
-				const record = await Subscription.findOneForGitHubInstallationId(
-					GITHUHB_INSTALLATION_ID,
-					GHEH_GITHUB_SERVER_APP_PK_ID_1
-				);
-				expect(record).toEqual(expect.objectContaining({
-					jiraHost,
-					jiraClientKey: "myClientKey_ghe_1",
-					gitHubInstallationId: GITHUHB_INSTALLATION_ID,
-					gitHubAppId: GHEH_GITHUB_SERVER_APP_PK_ID_1
-				}));
-			});
-		});
-		describe("getAllForInstallation", () => {
-			it("should only fetch cloud records when gitHubAppId not present", async () => {
-				const records = await Subscription.getAllForInstallation(
-					GITHUHB_INSTALLATION_ID
-				);
-				expect(records.length).toBe(1);
-				expect(records[0]).toEqual(expect.objectContaining({
-					jiraHost,
-					jiraClientKey: "myClientKey",
-					gitHubInstallationId: GITHUHB_INSTALLATION_ID,
-					gitHubAppId: null
-				}));
-			});
-			it("should fetch correct github server app", async () => {
-				const records = await Subscription.getAllForInstallation(
-					GITHUHB_INSTALLATION_ID,
-					GHEH_GITHUB_SERVER_APP_PK_ID_1
-				);
-				expect(records.length).toBe(1);
-				expect(records[0]).toEqual(expect.objectContaining({
-					jiraHost,
-					jiraClientKey: "myClientKey_ghe_1",
-					gitHubInstallationId: GITHUHB_INSTALLATION_ID,
-					gitHubAppId: GHEH_GITHUB_SERVER_APP_PK_ID_1
-				}));
-			});
-		});
-		describe("uninstall", () => {
-			it("should only uninstall cloud records when gitHubAppId not present", async () => {
-				await Subscription.uninstall({
-					host: jiraHost,
-					installationId: GITHUHB_INSTALLATION_ID
-				});
-				const [results] = await Subscription.sequelize!.query('select * from "Subscriptions"');
-				expect(results).toEqual([expect.objectContaining({
-					jiraClientKey: "myClientKey_ghe_1"
-				}), expect.objectContaining({
-					jiraClientKey: "myClientKey_ghe_2"
-				})]);
-			});
-			it("should uninstall correct github server app", async () => {
-				await Subscription.uninstall({
-					host: jiraHost,
-					installationId: GITHUHB_INSTALLATION_ID,
-					gitHubAppId: GHEH_GITHUB_SERVER_APP_PK_ID_1
-				});
-				const [results] = await Subscription.sequelize!.query('select * from "Subscriptions"');
-				expect(results).toEqual([expect.objectContaining({
-					jiraClientKey: "myClientKey"
-				}), expect.objectContaining({
-					jiraClientKey: "myClientKey_ghe_2"
-				})]);
-			});
-		});
+
+	describe.skip("syncStatusCounts", () => {
+		// TODO: add tests
```

Comment:
Do we have a task somewhere that keeps track of adding these tests?

Reply:
would be nice if we had a plugin that could take TODOs and add them to jira ;)

Reply:
but yeah, I was going to add some tasks to jira